libtcmu.c

```diff
@@ -765,6 +765,36 @@ uint32_t tcmu_get_dev_max_xfer_len(struct tcmu_device *dev)
 	return dev->max_xfer_len;
 }
 
+/**
+ * tcmu_set/get_dev_opt_unmap_gran - set/get device's optimal unmap granularity
```

Comment:
A single piece/chunk/unit of something is called a "grain", so you would use that instead of "gran" which just drops the rest of the letters on granularity.